lineQueueClient/src/Signup/Signup.js

```diff
@@ -1,17 +1,85 @@
-import React from 'react'
-import {
-  Text,
-  View
-} from 'react-native';
+import React, {Component} from 'react'
+import {Text, View, TextInput, StyleSheet, Image, Button} from 'react-native';
 
-const Signup = () => {
-	return (
-		<View>
-			<Text>
-				This is the signup page
-			</Text>
-		</View>
-	)
+export default class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: ''
+    };
+  }
+
+  render() {
+    return (
+      <View>
+        <View style={styles.containerBlack}>
+          <View style={{height: 20}} />
+          <Image style={styles.logo} source={require('../images/BezalelLogo.png')}></Image>
+          <View style={{height: 20}} />
+
+          <Text style={styles.title}>מערכת</Text>
+          <Text style={styles.title}>הזמנת תורים</Text>
+          <Text style={styles.title}>מחסן מרכזי</Text>
+
+          <View style={{height: 30}} />
+        </View>
+        <View style={styles.containerWhite}>
+          <View style={styles.containerWhite, {height: 50}} />
+          <Text style={styles.text}>
+
+            על מנת להזמין תור למחסן הציוד הזן את שמך:
+
+          </Text>
+          <TextInput style={styles.box} placeholder={"שם מלא"} onChangeText= { (text) => this.setState({text}) } value={this.state.text}/>
+          <View style={styles.border} />
+          <Button style={styles.button} title='send' color="#151515"
+          accessibilityLabel="Learn more about this purple button" />
+        </View>
+      </View>
+    )
+  }
 }
 
-export default Signup
+var styles = StyleSheet.create({
```

Comment:
Styles should go in a different file and imported into the component file. Style cascading should be considered for global variable to be extracted to global configuration.